build.gradle

```diff
@@ -203,14 +216,20 @@ task generateInstallers(type: Install4jTask, dependsOn: [renameShadowJar, copyJR
     }
 }
 
-task setRunnableLinux(dependsOn: [generateInstallers]) {
+task prepareInstallers(group: 'release', dependsOn: [generateInstallers]) {
     doLast {
-        def linuxFile = "TripleA_${version}_unix.sh"
-        ant.chmod(dir: "$buildDir/releases", perm: '+x', includes: linuxFile)
+        def releasesDir = "$buildDir/releases"
+
+        ant.chmod(dir: releasesDir, perm: '+x', includes: "TripleA_${version}_unix.sh")
```

Comment:
I'm wondering if this line is actually necessary.
Could you check if install4j generates files and marks them as executable?
Once they are uploaded the permission are reset anyways so the one who could benefit from this line is the developer.